vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_console_01_consoles.crd.yaml

```diff
@@ -77,6 +77,49 @@ spec:
                     - Azure
                     - ROSA
                     type: string
+                  capabilities:
+                    description: capabilities defines an array of capabilities that
+                      can be interacted with in the console UI. Each capability defines
+                      a visual state, by which it will get rendered. Available capabilities
+                      are LightspeedButton. Each of the available capabilities may
+                      appear only once in the list.
```

Comment:
Just a suggestion to the capabilities description:

> Each capability defines a visual state that can be interacted with the console to render in the UI. Available capabilities are LightspeedButton. Each of the available capabilities may appear only once in the list.

Also, one question: is the "may" in this sentence a possibility or that it "can" only appear once in the list? Just want to make sure.

Reply:
@opayne1 it can appear only once

Reply:
@jhadvig could we replace "may" with "can" to make it clear it can only appear once?